<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-SNIGSR3" ignored="0" name="Test" tests="6" failures="2" timestamp="12 Feb 2022 06:10:05 GMT" time="0.126" errors="0">
  <testcase name="test01" time="0.027" classname="com.jbk.DemoTest"/>
  <testcase name="test02" time="0.003" classname="com.jbk.DemoTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at com.jbk.DemoTest.test02(DemoTest.java:20)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
  <testcase name="test03" time="0.002" classname="com.jbk.DemoTest">
    <skipped/>
  </testcase> <!-- test03 -->
  <testcase name="test01" time="0.002" classname="com.jbk.DemoTest2"/>
  <testcase name="test02" time="0.01" classname="com.jbk.DemoTest2">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at com.jbk.DemoTest2.test02(DemoTest2.java:18)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
  <testcase name="test03" time="0.004" classname="com.jbk.DemoTest2">
    <skipped/>
  </testcase> <!-- test03 -->
</testsuite> <!-- Test -->
